src/OpenTelemetry.Api/README.md

```diff
@@ -358,12 +358,12 @@ to be associated with `Activity`. There is no `Status` class in .NET, and hence
 Example:
 
 ```csharp
-activity?.SetTag("otel.status_code", 2);
+activity?.SetTag("otel.status_code", StatusCode.Error);
```

Comment:
This section of the document is about instrumenting a library/app using just ActivitySource. i.e without any dependency on OpenTelemetry.API package. So we cannot use `StatusCode` here. Instead it must be the string "Error".

(once .NET support status natively, this should be done directly using the .net api, until then this is a workaround)

Reply:
Updated. What about the other references to enum I changed in this document? If I should also revert that, I am not sure what changes I should be writing about.

Reply:
This section cannot containing anything from OpenTelemetry packages. For status you can ask users to put "Error/Unset/Ok" strings.